### PR TITLE
fix(headers): add cache-control headers to api endpoints

### PR DESCRIPTION
**Pull request by jrgm**
r? - @seanmonstar, @g-k 

Discussion here: https://github.com/mozilla-services/cloudops-deployment/pull/334#issuecomment-272351058, and particularly here: https://github.com/mozilla-services/cloudops-deployment/pull/334#issuecomment-272502735, which I won't repeat here.

### CODE REVIEW

**Comment by g-k** [comment]
r+ this is a header we're looking for

I'll leave review of the implementation to sean.

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: b861d056f85724e412011794eadb4c3f11d5582d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-profile-server

---
This message was auto-generated by https://gitcop.com

**Comment by jrgm** [comment]
To be clear, I am setting cache headers on the public facing api (and tests for that). I'm not changing the internal-only daemons.

File: lib/server/web.js [hunk 1]
```diff
@@ -152,6 +152,15 @@ exports.create = function createServer() {
     }
     scopes = set(scopes);
   });
+
+  routes.forEach(function(route) {
+    if (route.config.cache === undefined) {
```
Comment (jrgm) [inline lib/server/web.js]: ... or should I be checking `!('cache' in route.config)` here instead?

Comment (seanmonstar) [inline lib/server/web.js]: They seem identical to me, and I don't have a preference of one over the other.

**Pull request merged**